getting_started/editor/unity_to_godot.rst

```diff
@@ -10,6 +10,10 @@ From Unity to Godot Engine
 This guide provides an overview of Godot Engine from the viewpoint of a Unity user,
 and aims to help you migrate your existing Unity experience into the world of Godot.
 
+.. note::
+
+   This article talks about older versions of Unity. Nestable prefabs ('Nested prefabs') were added to Unity 2018.3. Nestable prefabs are analogous to Godot's scenes, and allow a more Godot-like approach to scene organisation.
```

Comment:
IMO we should then either remove all talk about this or if there are notable differences point these out instead.

Not much use using this as an selling point otherwise, as it is just outdated information. Adding this note only makes it slightly less confusing, but not any more correct :)

Reply:
Agree. I'm thinking an entire re-write of this article might be necessary by someone who's had chance to get familiar with contemporary Unity workflow (with nested prefabs) and Godot.

Reply:
@mhilbrunner The purpose of this article is not to to list the selling points to a Unity user who desires to switch. To me it is primarily oriented to users who already made up their mind.
Thus it is not useless to point out the common points between Godot and Unity, but indeed it is useless to talk about them too much.